src/index.css

```diff
@@ -96,3 +96,8 @@ strong,
 b {
   font-weight: 700;
 }
+
+/* Fixes the smart banner being unclickable. We want this to rise above all app banners */
+.smartbanner--ios {
```

Comment:
I wonder if we have the same issue for android

Reply:
was wondering too but no one has reported and i'm unsure if there's going to be other side effects as a result (and i can't test, no android phone 😢 )

but very good point, will flag to @ruixhuang and @prashanDYDX

Reply:
on Android it's clickable but when it's visible it seems to push the nav bar off the screen